### ISSUE TITLE
Probably wrong display of the surface tag on roads

### ISSUE THREAD

**Issue by mfbehrens** [openstreetmap/iD#7466]
Hello,

I just added the surface tag to a living street and the street changed its style to the unpaved road look.
Is this correct or is this a bug?

Michael

**Comment by quincylvania** [comment]
@mfbehrens99 What `surface` value did you use? iD uses a narrow definition of "paved" which is more about smooth vs. bumpy. All values except these get the "unpaved" styling:

https://github.com/openstreetmap/iD/blob/5125c04a588d72683ab937949ea93bb0c819ab70/modules/osm/tags.js#L120-L131

**Comment by mfbehrens** [comment]
I mean this surface: 
wiki.openstreetmap.org/wiki/Key:surface

The way I mean is this here: 
openstreetmap.org/edit#map=21/50.00002/8.23741
openstreetmap.org/way/390473400

**Comment by quincylvania** [comment]
It looks like the tag in question is `surface=paving_stones`? This has come up before in #6691. The rationale is that most highways are considered have an asphalt-like surface by default in OSM, so it's helpful to style anything else differently.

However, it's true there's a real difference between paving stones and dirt. Perhaps we can consider a third style for intermediate-level pavement.

**Comment by mfbehrens** [comment]
But is paving stones not asphalt-like as well? After some rain there is no problem with mud or things like that. Same for the smoothness.

**Comment by quincylvania** [comment]
> Perhaps we can consider a third style for intermediate-level pavement.

I did this ☝️. I went with a subtle dash with the paved casing color that is meant to convey "not quite asphalt" while still being considerably better than earth or gravel.

<img width="1037" alt="Screen Shot 2020-03-26 at 7 21 47 PM" src="https://user-images.githubusercontent.com/2046746/77715182-1f5b2300-6f98-11ea-9045-eafc9dae1c63.png">

---

> But is paving stones not asphalt-like as well?

This definition no-doubt varies by culture. Here in the US, paving stones are extremely uncommon for public streets. Where they do exist, they vary considerably in smoothness quality. Thus, they're mentally distinct from what we'd typically think of as a paved road.
